app/javascript/src/components/OnboardingFollowTags.jsx

```diff
@@ -1,14 +1,15 @@
 import { h } from 'preact';
+import PropTypes from 'prop-types';
 import OnboardingTags from './OnboardingTags';
 
 // component of page 2
-const OnboardingFollowTags = ({ userData, allTags, followedTags, handleFollowTag }) => (
-  <OnboardingTags
-    userData={userData}
-    allTags={allTags}
-    followedTags={followedTags}
-    handleFollowTag={handleFollowTag}
-  />
+const OnboardingFollowTags = ({ allTags, handleFollowTag }) => (
+  <OnboardingTags allTags={allTags} handleFollowTag={handleFollowTag} />
 );
 
+OnboardingFollowTags.propTypes = {
+  allTags: PropTypes.objectOf().isRequired,
```

Comment:
Object of what? I think we should more specific here, shouldn't we?

Reply:
Hmm, i think i copied this from another component where it was the same. I'll try to investigate and see what a tag consists of...

Reply:
Yes, `OnboardingTags.jsx` has the same problem, could you fix both? Thanks!

Reply:
Created `./common-prop-types/tag-prop-types` and used `tagPropType` in OnboardingFollowTags.jsx, OnboardingTags.jsx and OnboardingSingleTag.jsx.